libs/api/domains/driving-license/src/lib/drivingLicense.service.ts

```diff
@@ -274,6 +275,10 @@ export class DrivingLicenseService {
     nationalId: User['nationalId'],
     input: NewDrivingLicenseInput,
   ): Promise<NewDrivingLicenseResult> {
+    // TODO: insert the following into body
```

Comment:
Is this something that needs immediate attention or just good to know in the future?

Reply:
Todo when the external api is updated